### ISSUE TITLE
Support setting names and descriptions of metadata from translations in the context locale

### ISSUE THREAD

**Issue by mseaton** [mekomsolutions/openmrs-module-initializer#248]
Currently, we have a setup where we have a single master set of metadata files (eg. a csv of encounter types) that is shared across multiple countries.  It is often requested that in each country, the actual name attached to the metadata (eg. encounter_type.name) be in the default locale used in that country.  This is useful, for example, when reports are run directly against the database, and the encounter_type.name is pulled in.  It is also useful if there are areas of the UI that do not use the localized display correctly.

One idea is to allow one to specify that the name and/or description of a given metadata row should be set based on the display in the default context locale.

**Comment by mks-d** [comment]
@mseaton is this still current or we can close this issue?

**Comment by mseaton** [comment]
There does appear to be an open PR around this @mks-d .  I can't remember the status:
https://github.com/mekomsolutions/openmrs-module-initializer/pull/249

**Comment by mks-d** [comment]
Yes 😞, I left a comment there too → https://github.com/mekomsolutions/openmrs-module-initializer/pull/249#discussion_r1658869248
